cmd/state/autoupdate.go

```diff
@@ -92,7 +92,7 @@ func autoUpdate(args []string, cfg *config.Instance, out output.Outputer) (bool,
 
 	code, err := relaunch(args)
 	if err != nil {
-		return true, &forwardExitError{code}
+		return true, errs.WrapExitCode(err, code, false)
```

Comment:
```suggestion
		return true, errs.WrapExitCode(err, code, true)
```

It looks like `forwardExitError` is silenced.

Also this change means we can drop `forwardExitError`.